MongoDb/Microsoft.DataTransfer.MongoDb.FunctionalTests/packages.config

```diff
@@ -1,8 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Castle.Core" version="3.3.3" targetFramework="net452" />
-  <package id="MongoDB.Bson" version="2.2.4" targetFramework="net452" />
-  <package id="MongoDB.Driver" version="2.2.4" targetFramework="net452" />
-  <package id="MongoDB.Driver.Core" version="2.2.4" targetFramework="net452" />
+  <package id="DnsClient" version="1.0.7" targetFramework="net452" />
```

Comment:
You need to review all new libraries you are adding/will be redistributing. Depending on the license, they may require you to update third-party notices as well as perform an IP scan on everything we plan to redistribute in pre-compiled form.

Reply:
Ack. I've submitted a request to perform an IP scan on the open source packages being pulled here. The only additional library added here for documentation purposes is DnsClient, which is a reference for MongoDb.

Reply:
The review has been completed, and all scan results have come back. I will take necessary actions to update licenses.